frontend/src/components/User/ChangeUserPassword.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { showValidationErrorToast, showServerErrorToast, showSuccessToast } from '../../utils/toast.js';
+import { updatePassword } from '../../api/UserApi.js';
+
+const ChangeUserPassword = ({ user }) => {
+  const [currentPassword, setCurrentPassword] = useState('');
+  const [newPassword, setNewPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  const navigate = useNavigate();
+
+  const handleUpdateClick = async (e) => {
+    e.preventDefault();
+    updatePassword(user.id, currentPassword, newPassword, confirmPassword)
+      .then(() => {
+        navigate(-1);
```

Comment:
What is this supposed to do? Perhaps put a comment here?

Reply:
It redirects user back to the previous page (user-profile) if the password is updated successfully. I have added comment in the next commit